### ISSUE TITLE
Support other distros

### ISSUE THREAD

**Issue by Jacoby6000** [kicker22004/ECO_LINUX_SERVER_MANAGER#5]
As far as I can tell, the only thing stopping this from using other distros is the fact that the deps stuff uses apt.  Would you accept a PR which allows checking linux distro and following the proper installation procedures based on it?

**Comment by kicker22004** [comment]
Yeah it's in the next release, I've just been really busy and had no time
to write it. (2 jobs)
